package-lock.json

```diff
@@ -12100,238 +12100,273 @@
     }
   },
   "dependencies": {
+    "@ampproject/remapping": {
```

Comment:
Looks great! :100: 

For information:
We have moved from `npm` to `yarn`.
You should get new changes from upstream, and re-create lock files using yarn (`yarn.lock`).

**TODOs:**
- [ ] Merge incoming upstream changes
- [ ] Fix suggested changes
Should be good after that!